client/components/Related-items-and-comparisons/Card.jsx

```diff
@@ -10,18 +10,6 @@ const Card = (props) => {
 
   if(props.relatedProductID !== undefined) {
```

Comment:
is this if statement breaking the app? Looks like the closing curly bracket is now commented out on line 23